src/pages/_error.tsx

```diff
@@ -20,27 +23,40 @@ const Error: NextPageWithLayout<Props> = ({ statusCode }: Props) => {
 
   return (
     <>
+      <h1>500</h1>
```

Comment:
Suggestion: use `statusCode` in the header if it's available.

```suggestion
      <h1>{statusCode ? statusCode : 500}</h1>
```